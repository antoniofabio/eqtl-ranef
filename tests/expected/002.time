	Command being timed: "./test-002.sh"
	User time (seconds): 181.87
	System time (seconds): 1.19
	Percent of CPU this job got: 21%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 14:18.15
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 809312
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 103285
	Voluntary context switches: 5908
	Involuntary context switches: 229237
	Swaps: 0
	File system inputs: 0
	File system outputs: 7672
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
