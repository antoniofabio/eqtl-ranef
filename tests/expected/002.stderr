F : 'data.frame':	100 obs. of  2 variables:
 $ sampleID: Factor w/ 100 levels "sample1","sample10",..: 1 13 24 35 46 57 68 79 90 2 ...
 $ F1      : num  0.114 0.622 0.609 0.623 0.861 ...
R : 'data.frame':	100 obs. of  2 variables:
 $ sampleID: Factor w/ 100 levels "sample1","sample10",..: 1 13 24 35 46 57 68 79 90 2 ...
 $ R1      : Factor w/ 2 levels "1","2": 1 2 1 2 1 2 1 2 1 2 ...
X : 'data.frame':	200 obs. of  3 variables:
 $ regressorID: Factor w/ 2 levels "regressor1","regressor2": 1 1 1 1 1 1 1 1 1 1 ...
 $ sampleID   : Factor w/ 100 levels "sample1","sample10",..: 1 2 3 4 5 6 7 8 9 10 ...
 $ value      : num  0.5449 0.7624 0.7807 0.0215 0.6784 ...
Y : 'data.frame':	100000 obs. of  3 variables:
 $ outcomeID: Factor w/ 1000 levels "outcome1","outcome10",..: 1 1 1 1 1 1 1 1 1 1 ...
 $ sampleID : Factor w/ 100 levels "sample1","sample10",..: 1 2 3 4 5 6 7 8 9 10 ...
 $ value    : num  -1.806 0.0808 -0.3445 0.3169 0.2959 ...
option_list : List of 6
 $ :Formal class 'OptionParserOption' [package "optparse"] with 8 slots
 $ :Formal class 'OptionParserOption' [package "optparse"] with 8 slots
 $ :Formal class 'OptionParserOption' [package "optparse"] with 8 slots
 $ :Formal class 'OptionParserOption' [package "optparse"] with 8 slots
 $ :Formal class 'OptionParserOption' [package "optparse"] with 8 slots
 $ :Formal class 'OptionParserOption' [package "optparse"] with 8 slots
opts : List of 7
 $ regressors: num 2
 $ outcomes  : num 1000
 $ samples   : num 100
 $ fixef     : num 1
 $ output    : chr "/tmp/tmp.rBrHBS9fU0"
 $ seed      : num 1234
 $ help      : logi FALSE
outcomesIDs :  chr [1:1000] "outcome1" "outcome2" "outcome3" "outcome4" ...
parser : Formal class 'OptionParser' [package "optparse"] with 4 slots
regressorsIDs :  chr [1:2] "regressor1" "regressor2"
samplesIDs :  chr [1:100] "sample1" "sample2" "sample3" "sample4" "sample5" ...
chromosomeLength = 2000
  reporterID chromosome start end
1   outcome1          1     1   2
2   outcome2          1     3   4
3   outcome3          1     5   6
4   outcome4          1     7   8
5   outcome5          1     9  10
6   outcome6          1    11  12
       SNPID chromosome position
1 regressor1          1        1
2 regressor2          1     2001
analysis completed.
List of 13
 $ regressors           : chr "/tmp/tmp.rBrHBS9fU0/regressors"
 $ outcomes             : chr "/tmp/tmp.rBrHBS9fU0/outcomes"
 $ fixef                : chr "/tmp/tmp.rBrHBS9fU0/fixef"
 $ ranef                : chr "/tmp/tmp.rBrHBS9fU0/ranef"
 $ genespos             : chr "/tmp/tmp.rBrHBS9fU0/genespos.sqlite"
 $ snpspos              : chr "/tmp/tmp.rBrHBS9fU0/snpspos.sqlite"
 $ cis-window           : num 5e+05
 $ trans-pvalue         : num 0.001
 $ cis-pvalue           : num 0.1
 $ cores                : num 1
 $ max-regressions      : int 10000
 $ regressors-chunk-size: int 1000
 $ help                 : logi FALSE
reading outcomes data...
done:
'data.frame':	100000 obs. of  3 variables:
 $ reporterID: chr  "outcome1" "outcome1" "outcome1" "outcome1" ...
 $ sampleID  : chr  "sample1" "sample10" "sample100" "sample11" ...
 $ expression: num  -1.806 0.0808 -0.3445 0.3169 0.2959 ...
reading fixed effects data...
done:
'data.frame':	100 obs. of  2 variables:
 $ sampleID: chr  "sample1" "sample2" "sample3" "sample4" ...
 $ F1      : num  0.114 0.622 0.609 0.623 0.861 ...
reading random effects data...
done:
'data.frame':	100 obs. of  2 variables:
 $ sampleID: chr  "sample1" "sample2" "sample3" "sample4" ...
 $ R1      : int  1 2 1 2 1 2 1 2 1 2 ...
merging covariates data...
'data.frame':	100 obs. of  3 variables:
 $ sampleID: chr  "sample1" "sample10" "sample100" "sample11" ...
 $ F1      : num  0.114 0.514 0.742 0.694 0.545 ...
 $ R1      : int  1 2 2 1 2 1 2 1 2 1 ...
merging outcomes data...
'data.frame':	100000 obs. of  5 variables:
 $ sampleID  : chr  "sample1" "sample10" "sample100" "sample11" ...
 $ reporterID: chr  "outcome1" "outcome1" "outcome1" "outcome1" ...
 $ expression: num  -1.806 0.0808 -0.3445 0.3169 0.2959 ...
 $ F1        : num  0.114 0.514 0.742 0.694 0.545 ...
 $ R1        : int  1 2 2 1 2 1 2 1 2 1 ...
model formula:
expression ~ alleleCopyNumber + F1 + (1 | R1)

=> main loop started on Fri May 23 10:04:16 2014

no more lines in input: terminating...

=> main loop completed on Fri May 23 10:17:51 2014

2 regressors processed in 13.57945 mins
 reading input: 0.06233764 secs
 joining data: 9.733643 secs
 processing: 13.37588 mins
 filtering output: 0.198689 secs
 writing output: 0.04572439 secs
 garbage collection: 1.962326 secs
analysis completed.
